Installer doesn't include external module

After installing the package, the extension doesn't load correctly. It throws an error when activating extension because it cannot find module 'xml2js'. More info below:
````
[2020-02-06 12:35:36.221] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/mbc/.vscode/extensions/plastic-scm.plastic-scm-0.0.1/out/extension.js
[2020-02-06 12:35:36.239] [exthost] [error] Activating extension plastic-scm.plastic-scm failed due to an error:
[2020-02-06 12:35:36.242] [exthost] [error] Error: Cannot find module 'xml2js'
Require stack:
- c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\cm\commands\status\statusParser.js
- c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\cm\commands\status\status.js
- c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\cm\commands\index.js
- c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\plasticScm.js
- c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\extension.js
- c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js
- c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-fork.js
	at Function.Module._resolveFilename (internal/modules/cjs/loader.js:627:15)
	at Function.Module._load (internal/modules/cjs/loader.js:531:27)
	at Function.t._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:818:851)
	at Function.n._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:789:164)
	at Function.i._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:785:618)
	at Module.require (internal/modules/cjs/loader.js:685:19)
	at n (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:524)
	at Object.<anonymous> (c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\cm\commands\status\statusParser.js:14:16)
	at Module.o._compile (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:749)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:798:10)
	at Module.load (internal/modules/cjs/loader.js:645:32)
	at Function.Module._load (internal/modules/cjs/loader.js:560:12)
	at Function.t._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:818:851)
	at Function.n._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:789:164)
	at Function.i._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:785:618)
	at Module.require (internal/modules/cjs/loader.js:685:19)
	at n (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:524)
	at Object.<anonymous> (c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\cm\commands\status\status.js:13:24)
	at Module.o._compile (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:749)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:798:10)
	at Module.load (internal/modules/cjs/loader.js:645:32)
	at Function.Module._load (internal/modules/cjs/loader.js:560:12)
	at Function.t._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:818:851)
	at Function.n._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:789:164)
	at Function.i._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:785:618)
	at Module.require (internal/modules/cjs/loader.js:685:19)
	at n (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:524)
	at Object.<anonymous> (c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\cm\commands\index.js:5:16)
	at Module.o._compile (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:749)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:798:10)
	at Module.load (internal/modules/cjs/loader.js:645:32)
	at Function.Module._load (internal/modules/cjs/loader.js:560:12)
	at Function.t._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:818:851)
	at Function.n._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:789:164)
	at Function.i._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:785:618)
	at Module.require (internal/modules/cjs/loader.js:685:19)
	at n (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:524)
	at Object.<anonymous> (c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\plasticScm.js:14:20)
	at Module.o._compile (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:749)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:798:10)
	at Module.load (internal/modules/cjs/loader.js:645:32)
	at Function.Module._load (internal/modules/cjs/loader.js:560:12)
	at Function.t._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:818:851)
	at Function.n._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:789:164)
	at Function.i._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:785:618)
	at Module.require (internal/modules/cjs/loader.js:685:19)
	at n (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:524)
	at Object.<anonymous> (c:\Users\mbc\.vscode\extensions\plastic-scm.plastic-scm-0.0.1\out\extension.js:5:22)
	at Module.o._compile (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:15:749)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:798:10)
	at Module.load (internal/modules/cjs/loader.js:645:32)
	at Function.Module._load (internal/modules/cjs/loader.js:560:12)
	at Function.t._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:818:851)
	at Function.n._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:789:164)
	at Function.i._load (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:785:618)
	at Module.require (internal/modules/cjs/loader.js:685:19)
	at require (internal/modules/cjs/helpers.js:16:16)
	at Function.s [as __$__nodeRequire] (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:32:963)
	at d._loadCommonJSModule (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:790:289)
	at d._doActivateExtension (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:694:547)
	at d._activateExtension (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:693:582)
	at Object.actualActivateExtension (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:691:534)
	at d._activateExtension (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:517:21)
	at c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:516:632
	at Array.map (<anonymous>)
	at d._activateExtensions (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:516:620)
	at d.activateByEvent (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:514:413)
	at d._activateByEvent (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:692:282)
	at d._handleEagerExtensions (c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:696:122)
	at c:\Users\mbc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:699:532
	at processTicksAndRejections (internal/process/task_queues.js:89:5)
````

--- a/package.json
+++ b/package.json
@@ -63,6 +63,11 @@
     "test": "node ./out/test/runTest.js",
     "deploy": "vsce publish"
   },
+  "dependencies":{
+    "uuid": "^3.3.3",
+    "byline": "^5.0.0",
+    "xml2js": "^0.4.23"
+  },
   "devDependencies": {
     "@types/byline": "^4.2.31",
     "@types/chai": "^4.2.7",
@@ -71,8 +76,7 @@
     "@types/node": "^12.11.7",
     "@types/uuid": "^3.4.6",
     "@types/xml2js": "^0.4.5",
-    "@types/vscode": "^1.41.0",
-    "byline": "^5.0.0",
+    "@types/vscode": "^1.19.0",
     "chai": "^4.2.0",
     "glob": "^7.1.5",
     "mocha": "^6.2.2",
@@ -80,9 +84,7 @@
     "tslint": "^5.20.1",
     "typemoq": "^2.1.0",
     "typescript": "^3.6.4",
-    "uuid": "^3.3.3",
     "vsce": "^1.71.0",
-    "vscode-test": "^1.2.2",
-    "xml2js": "^0.4.23"
+    "vscode-test": "^1.2.2"
   }
 }

